rust/noosphere-sphere/src/context.rs

```diff
@@ -543,10 +543,46 @@ pub mod tests {
         assert_eq!(name.as_str(), "c");
     }
 
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+    #[cfg_attr(not(target_arch = "wasm32"), tokio::test)]
+    async fn it_can_traverse_a_sequence_of_petnames_one_at_a_time() {
```

Comment:
Related to #320?

Reply:
Yes, I wrote this change while investigating that and since I had the test I included here.